### PR TITLE
Bug 1150322 - make the seek bar behave a little better for songs of infi...

### PR DESCRIPTION
**Pull request by davidflanagan**
...nite duration

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ddc34689b7d94545d16076084311d85588f0bc6c

File: apps/music/js/ui/views/player_view.js [hunk 1]
```diff
@@ -763,27 +763,14 @@ var PlayerView = {
       this.audio.currentTime = Math.floor(seekTime);
     }
 
-    var startTime = this.audio.startTime;
```
Comment (hfiguiere) [inline apps/music/js/ui/views/player_view.js]: I had a patch to remove this because ````startTime```` doesn't exist. :+1:

Comment (davidflanagan) [inline apps/music/js/ui/views/player_view.js]: I could swear it used to exist. Maybe it was in a draft spec and then removed? It was supposed to be for things like live streams. If you started streaming at 15:02, then this gave you a way to display that number as the start point. But it does appear completely undocumented now.

Comment (hfiguiere) [inline apps/music/js/ui/views/player_view.js]: and is ````undefinited````

**Pull request merged**